### PR TITLE
Split the fxdependent package on Windows into two packages

### LINKED ISSUE #10019: Framework dependent package for 7.0.0-preview1 does not work on Windows ARM and ARM64
See https://github.com/dotnet/dotnet-docker/pull/1147

# Steps to reproduce

Start fxdependent powershell on win-arm or win-arm64

 We have the following logic for deciding the SDK we want to use. 

```
<Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk" Condition="'$(RuntimeIdentifier)' == 
'win-arm' or '$(RuntimeIdentifier)' == 'win-arm64'" />
  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk" Condition="'$(RuntimeIdentifier)' == 'win-arm' or '$(RuntimeIdentifier)' == 'win-arm64'" />

  <Import Project="Sdk.props" Sdk="Microsoft.NET.Sdk.WindowsDesktop" Condition="'$(RuntimeIdentifier)' != 'win-arm' and '$(RuntimeIdentifier)' != 'win-arm64'" />
  <Import Project="Sdk.targets" Sdk="Microsoft.NET.Sdk.WindowsDesktop" Condition="'$(RuntimeIdentifier)' != 'win-arm' and '$(RuntimeIdentifier)' != 'win-arm64'" />
```

When we build a framework dependent package, we have no specified RuntimeIdentifier, hence we select `Microsoft.NET.Sdk.WindowsDesktop`. We can only fix this in the next release for PowerShell.

# Expected behavior

```none
Powershell starts
```

# Actual behavior

```none
Tries to load Microsoft.NET.Sdk.WindowsDesktop which is not supported.
```

# Environment data

```none

```

### PR DESCRIPTION
**Pull request by adityapatwardhan**
# PR Summary

The `Microsoft.NET.Sdk.WindowsDesktop` SDK is not supported on ARM and ARM64. We need fxdependent package targetting both `Microsoft.NET.Sdk.WindowsDesktop` and `Microsoft.NET.Sdk` for global tool. Hence, we create two packages now.

Fixes #10019

## PR Checklist

- [ ] [PR has a meaningful title](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
    - Use the present tense and imperative mood when describing your changes
- [ ] [Summarized changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] [Make sure all `.h`, `.cpp`, `.cs`, `.ps1` and `.psm1` files have the correct copyright header](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
- [ ] This PR is ready to merge and is not [Work in Progress](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---work-in-progress).
    - If the PR is work in progress, please add the prefix `WIP:` or `[ WIP ]` to the beginning of the title (the `WIP` bot will keep its status check at `Pending` while the prefix is present) and remove the prefix when the PR is ready.
- **[Breaking changes](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#making-breaking-changes)**
    - [ ] None
    - **OR**
    - [ ] [Experimental feature(s) needed](https://github.com/MicrosoftDocs/PowerShell-Docs/blob/staging/reference/6/Microsoft.PowerShell.Core/About/about_Experimental_Features.md)
        - [ ] Experimental feature name(s): 
- **User-facing changes**
    - [ ] Not Applicable
    - **OR**
    - [ ] [Documentation needed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#pull-request---submission)
        - [ ] Issue filed: 
- **Testing - New and feature**
    - [ ] N/A or can only be tested interactively
    - **OR**
    - [ ] [Make sure you've added a new test if existing tests do not effectively test the code changed](https://github.com/PowerShell/PowerShell/blob/master/.github/CONTRIBUTING.md#before-submitting)

### CODE REVIEW

**Review by TravisEz13 — APPROVED** [review]

**Review by TravisEz13 — APPROVED** [review]

File: build.psm1 [hunk 1]
```diff
@@ -220,6 +220,7 @@ function Start-PSBuild {
         # If this parameter is not provided it will get determined automatically.
```
Comment (PoshChan) [inline build.psm1]: @adityapatwardhan, your last commit had 1 failures in `PowerShell-CI-windows`
Set/New/Remove-Service cmdlet tests.Get-Service can get the 'UserName' of a service
```powershell
Expected a value, but got $null or empty.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Management\Set-Service.Tests.ps1: line 304
304:             $service | Should -Not -BeNullOrEmpty
```

**Comment by adityapatwardhan** [comment]
@PoshChan rebuild all

**Comment by PoshChan** [comment]
@adityapatwardhan, all requests start with the magic word: `Please`

Commands available in this repo for you:
  - `retry <target>` this will attempt to retry only the failed jobs for the target pipeline
  - `rebuild <target>` this will perform a complete rebuild of the target pipeline, `rerun` can be used in place of `rebuild`
    Supported values for \<target\> which can be a comma separated list are: `static`,`windows`,`macos`,`linux`,`all`
  - `get failures` this will attempt to get the latest failures for all of the target pipelines
  - `remind me in <value> <units>` this will create a reminder that will be posted after the specified duration
    \<value\> is a number, and \<units\> can be `minutes`, `hours`, or `days` (singular or plural)

**Comment by adityapatwardhan** [comment]
@PoshChan please rebuild all

**Comment by PoshChan** [comment]
@adityapatwardhan, successfully started rebuild of `PowerShell-CI-static-analysis, PowerShell-CI-Windows, PowerShell-CI-macOS, PowerShell-CI-Linux`

**Pull request merged**